.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and test
+
+on:
+  push:
+    branches: ['**', '!backport/**']
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches: ['**']
+    paths-ignore:
+      - '**/*.md'
+
+jobs:
+  build-lint-test:
+    runs-on: ${{ matrix.os }}
+    name: Build and Verify
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'yarn'
+
+      - name: Setup Yarn
+        run: |
+          npm uninstall -g yarn
+          npm i -g yarn@1.22.10
+          yarn install --frozen-lockfile
```

Comment:
This seems unnecessary, see https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data. The previous action should already install and cache yarn correctly. I havent validated this, but this seems to be correct based on the documentation for `setup-node`

Reply:
Reading through the docs and a little bit of the source code for the action, it looks like it essentially is just a shorthand to use the [`actions/cache`](https://github.com/actions/cache/blob/main/examples.md#node---npm) action on `node_modules` and `yarn.lock`, meaning that if we update dependencies or change dependencies, those changes wouldn't be reflected in the workflow.